XMonad/Hooks/UrgencyHook.hs

```diff
@@ -254,7 +254,13 @@ minutes secs = secs * 60
 -- Use a variation of this in your config just as you use a variation of
 -- 'def' for your xmonad definition.
 urgencyConfig :: UrgencyConfig
-urgencyConfig = UrgencyConfig { suppressWhen = Visible, remindWhen = Dont }
+urgencyConfig = def
+
+-- | The default 'UrgencyConfig'. suppressWhen = Visible, remindWhen = Dont.
+-- Use a variation of this in your config just as you use a variation of
```

Comment:
We probably want to at least put these in a monospace font (between `@`s)

Reply:
Wrapped `suppressWhen = Visible, remindWhen = Dont` in `@`s.
Is this what you had in mind?

Reply:
Yes exactly; Haddock will print this in a monospaced font, which I think looks nicer for code